src/schema/nmdc.yaml

```diff
@@ -362,6 +398,10 @@ classes:
         structured_pattern:
           syntax: "{id_nmdc_prefix}:dobj-{id_shoulder}-{id_blade}$"
           interpolated: true
+      was_generated_by:
+        structured_pattern:
+          syntax: "{id_nmdc_prefix}:(wfmag|wfmb|wfmgan|wfmgas|wfmsa|wfmp|wfmt|wfmtan|wfmtas|wfnom|wfrbt|wfrqc|wf|omprc)-{id_shoulder}-{id_blade}{id_version}$"
```

Comment:
'wf' should be removed, WorkflowExeuctionActivity is abstract. id_version should be allowed on workflow activity subclasses but not OmicsProcessing records so I think this should be re-written like this
syntax: "{id_nmdc_prefix}:(wfmag|wfmb|wfmgan|wfmgas|wfmsa|wfmp|wfmt|wfmtan|wfmtas|wfnom|wfrbt|wfrqc)-{id_shoulder}-{id_blade}{id_version}$ | {id_nmdc_prefix}:omprc-{id_shoulder}-{id_blade}$"